test/UnitTests/MSTest.Analyzers.UnitTests/PreferDisposeOverTestCleanupAnalyzerTests.cs

```diff
@@ -45,53 +45,142 @@ public ValueTask DisposeAsync()
             }
             """;
 
-        await VerifyCS.VerifyAnalyzerAsync(code);
+        await VerifyCS.VerifyCodeFixAsync(code, code);
     }
 
     public async Task WhenTestClassHasTestCleanup_Diagnostic()
     {
         string code = """
             using Microsoft.VisualStudio.TestTools.UnitTesting;
-
+            
             [TestClass]
             public class MyTestClass
             {
                 [TestCleanup]
                 public void [|MyTestCleanup|]()
+                {
+                    int x = 1;
+                }
+            }
+            """;
+        string fixedCode = """
+            using System;
+            using Microsoft.VisualStudio.TestTools.UnitTesting;
+            
+            [TestClass]
+            public class MyTestClass
+            : IDisposable
+            {
+                public void Dispose()
```

Comment:
should it be virtual to allow override? https://learn.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose

Reply:
but the usual fix for implementing the interface gives the not virtual one